packages/automator/index.tsx

```diff
@@ -8,6 +8,8 @@ import {
   stepUntilConvergence,
 } from "@penrose/core";
 import { renderArtifacts } from "./artifacts";
+import { getListOfStagedStates } from "../core/src/renderer/Staging";
```

Comment:
Kinda surprised that this worked. Export your functions in `index.tsx` of `core` and import it here from `@penrose/core`. These are separate packages, so relative paths won't work in general.